addons/payment/views/payment_form_templates.xml

```diff
@@ -408,37 +407,66 @@
 
     <template id="payment.no_pms_available_warning">
         <div class="alert alert-warning mt-2">
-            <strong>No suitable payment option could be found.</strong>
-            <!-- === Availability report button (for debug) === -->
-            <t t-call="payment.availability_report_button"/>
+            <strong>No payment method available</strong>
             <br/>
-            If you believe that it is an error, please contact the website administrator.
+            <t t-if="request.env.is_system()">
+                <p t-if="providers_sudo">
+                    None is configured for
+                    <t t-out="request.env.company.country_id.name"/>,
+                    <t t-out="request.env.company.currency_id.name"/>.
+                </p>
+                <p t-else="">
+                    No payment providers are configured.
+                </p>
+            </t>
+            <t t-else="">
+                If you believe that it is an error, please contact the website administrator.
+            </t>
             <div class="mt-2" groups="base.group_system">
                 <a
-                    t-if="request.env.company.country_id.is_stripe_supported_country"
+                    t-if="request.env.company.country_id.is_stripe_supported_country
+                          and not providers_sudo"
                     name="activate_stripe"
                     href="/odoo/action-payment.action_activate_stripe"
                     role="button"
                     class="btn btn-primary"
                 > ACTIVATE STRIPE </a>
                 <a
+                    t-if="availability_report"
+                    role="button"
+                    class="btn-link alert-warning"
+                    data-bs-toggle="collapse"
+                    href="#payment_availability_report"
+                >
+                    <strong><i class="fa fa-eye"/> Show debug report</strong>
+                </a>
+                <a
+                    t-if="not providers_sudo"
                     role="button"
                     type="action"
                     class="btn-link alert-warning ps-2"
```

Comment:
Still [too much padding](https://github.com/odoo/odoo/pull/165203#discussion_r1721712539) when only "Payment Providers" is shown.

![image](https://github.com/user-attachments/assets/f642116e-0221-44af-96e9-410e32772057)

I'd add `pe-2` to previous buttons instead so that the padding is there only when buttons are shown.

Reply:
Done the changes 
I had use ps-2 because when there is ''ACTIVATE STRIPE" button it will stick to the "payment provider" button.